dev/deps/spark-deps-hadoop-2-hive-2.3

```diff
@@ -200,24 +200,25 @@ metrics-jmx/4.2.13//metrics-jmx-4.2.13.jar
 metrics-json/4.2.13//metrics-json-4.2.13.jar
 metrics-jvm/4.2.13//metrics-jvm-4.2.13.jar
 minlog/1.3.0//minlog-1.3.0.jar
-netty-all/4.1.84.Final//netty-all-4.1.84.Final.jar
-netty-buffer/4.1.84.Final//netty-buffer-4.1.84.Final.jar
-netty-codec-http/4.1.84.Final//netty-codec-http-4.1.84.Final.jar
-netty-codec-http2/4.1.84.Final//netty-codec-http2-4.1.84.Final.jar
-netty-codec-socks/4.1.84.Final//netty-codec-socks-4.1.84.Final.jar
-netty-codec/4.1.84.Final//netty-codec-4.1.84.Final.jar
-netty-common/4.1.84.Final//netty-common-4.1.84.Final.jar
-netty-handler-proxy/4.1.84.Final//netty-handler-proxy-4.1.84.Final.jar
-netty-handler/4.1.84.Final//netty-handler-4.1.84.Final.jar
-netty-resolver/4.1.84.Final//netty-resolver-4.1.84.Final.jar
-netty-transport-classes-epoll/4.1.84.Final//netty-transport-classes-epoll-4.1.84.Final.jar
-netty-transport-classes-kqueue/4.1.84.Final//netty-transport-classes-kqueue-4.1.84.Final.jar
-netty-transport-native-epoll/4.1.84.Final/linux-aarch_64/netty-transport-native-epoll-4.1.84.Final-linux-aarch_64.jar
-netty-transport-native-epoll/4.1.84.Final/linux-x86_64/netty-transport-native-epoll-4.1.84.Final-linux-x86_64.jar
-netty-transport-native-kqueue/4.1.84.Final/osx-aarch_64/netty-transport-native-kqueue-4.1.84.Final-osx-aarch_64.jar
-netty-transport-native-kqueue/4.1.84.Final/osx-x86_64/netty-transport-native-kqueue-4.1.84.Final-osx-x86_64.jar
-netty-transport-native-unix-common/4.1.84.Final//netty-transport-native-unix-common-4.1.84.Final.jar
-netty-transport/4.1.84.Final//netty-transport-4.1.84.Final.jar
+netty-all/4.1.86.Final//netty-all-4.1.86.Final.jar
+netty-buffer/4.1.86.Final//netty-buffer-4.1.86.Final.jar
+netty-codec-http/4.1.86.Final//netty-codec-http-4.1.86.Final.jar
+netty-codec-http2/4.1.86.Final//netty-codec-http2-4.1.86.Final.jar
+netty-codec-socks/4.1.86.Final//netty-codec-socks-4.1.86.Final.jar
+netty-codec/4.1.86.Final//netty-codec-4.1.86.Final.jar
+netty-common/4.1.86.Final//netty-common-4.1.86.Final.jar
+netty-handler-proxy/4.1.86.Final//netty-handler-proxy-4.1.86.Final.jar
+netty-handler-ssl-ocsp/4.1.86.Final//netty-handler-ssl-ocsp-4.1.86.Final.jar
```

Comment:
`netty-handler-ssl-ocsp`, seems a new dependency. Does Spark really need it?

Reply:
I don't think so, but maven did add this when I run `./dev/test-dependencies.sh --replace-manifest`

Reply:
Is it easy to manually exclude?

Reply:
I don't know. I don't think we should do it. 
Let it be there.

Reply:
If we are sure that we do not need this new jar, I suggest remove it, WDYT @HyukjinKwon

Reply:
https://github.com/netty/netty/issues/13003 they have done this on purpose

Reply:
just to add it her 
https://github.com/apache/spark/blob/ae5c9eb6dd241e7916101c302f1fa84a8fe08141/pom.xml#L894

Reply:
@LuciferYang Thank you :) 
I have now added netty-handler-ssl-ocsp to exclude list. and run `./dev/test-dependencies.sh --replace-manifest`